### PR TITLE
xz 5.4.2

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/xz.rb [hunk 1]
```diff
@@ -4,11 +4,11 @@ class Xz < Formula
   desc "General-purpose data compression with high compression ratio"
   homepage "https://tukaani.org/xz/"
   # The archive.org mirror below needs to be manually created at `archive.org`.
```
Comment (carlocab) [inline Formula/xz.rb:6]: Don't forget to do this!

Comment (p-linnane) [reply Formula/xz.rb:6]: I took care of it and it's now live.

Comment (carlocab) [reply Formula/xz.rb:6]: Thanks!

**Review by carlocab — APPROVED** [review]
Failures look pre-existing.

**Comment by Porkepix** [comment]
Linux job seems to be completely stuck (also, there are at least 3 others PR waiting for the self-hosted runner on Linux, which seems to only offer a single slot…)

**Pull request closed without merging**